acquia/acquia_cms
# ACMS-1020: Update site studio to 6.8.x

**Motivation**

Fixes #[ACMS-1020](https://backlog.acquia.com/browse/ACMS-1020)
As a developer
I want to use latest version of site studio (6.8.x) with acquia CMS
So that all components/helper are updated and work well with it.

**Proposed changes**

Update site studio package as per 6.8.x standard

**Alternatives considered**

NA

**Testing steps**


- Install site with site studio key preset
- Verify each components/helper and make sure everything is working as expected
- Visit package listing page and assure all packages are present.
- Run test suits to check nothing breaks
- Enable starter module
- Perform smoke test and make sure everything is align 

**Merge requirements**
- [_Enhancement_] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
